fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsBottomSheet.kt

```diff
@@ -135,14 +135,18 @@ internal fun TranslationsDialog(
     onSettingClicked: () -> Unit,
     onLearnMoreClicked: () -> Unit,
     onTranslateButtonClick: () -> Unit,
+    onNotNowButtonClick: () -> Unit,
```

Comment:
```suggestion
    onTranslateButtonClicked: () -> Unit,
    onNotNowButtonClicked: () -> Unit,
```
This is just to align with how we already named things here.

Reply:
Done